src/base/memory_monitor_sr.cpp

```diff
@@ -92,6 +92,77 @@ namespace cubmem
 
     return filecopy + ':' + std::to_string (line);
   }
+
+  void memory_monitor::add_stat (char *ptr, const size_t size, const char *file, const int line)
```

Comment:
```suggestion
  void
  memory_monitor::add_stat (char *ptr, const size_t size, const char *file, const int line)
```

Hmm, the indentation for most C++ codes in Cubrid is configured like this.
It can make too much changes..

Reply:
I'll change it with future refactoring issue :) Thanks.